frontend/apps/filemanager/filemanager.lua

```diff
@@ -434,7 +434,7 @@ function FileManager:setupLayout()
     if Device:hasKeys() then
         self.key_events.Home = { {"Home"}, doc = "go home" }
         -- Override the menu.lua way of handling the back key
-        self.file_chooser.key_events.Back = { {"Back"}, doc = "go back" }
+        self.file_chooser.key_events.Back = { {Device.input.group.Back}, doc = "go back" }
```

Comment:
Is that supposed to be another table in the table?

Reply:
I guess so. I haven't looked at how it is handled (I guess `if type(thing) == table then do something elif type(thing) == string than do otherthing`), but that's how all other Input.group.PgFw et al. do - so I didn't need to check :)
But ok, checked, and discovered this module I have never ever meet !:
https://github.com/koreader/koreader/blob/d7c00c586f88910dd532d6a59ab2abb48f7b958e/frontend/device/key.lua#L44-L72

Reply:
It just looks like a redundant extra table because the group is a table just like the table you're replacing.

Reply:
I agree it feels odd - but all other group.Stuff are done that way. So better not adding inconsistencies - and not fixing what's not broken :/

Reply:
Alright.